Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -574,10 +553,16 @@ PhysicalParticleContainer::AddPlasma (int lev, RealBox part_realbox)
     Real scale_fac;
 #if AMREX_SPACEDIM==3
     scale_fac = dx[0]*dx[1]*dx[2]/num_ppc;
-    amrex::Print() << " scale_fac " << scale_fac << "\n";
 #elif AMREX_SPACEDIM==2
     scale_fac = dx[0]*dx[1]/num_ppc;
 #endif
+#ifdef PULSAR
+    // modifying particle weight to achieve fractional injection
+    if (PulsarParm::ModifyParticleWtAtInjection == 1) {
+        scale_fac = scale_fac*PulsarParm::Ninj_fraction;
+        amrex::Print() << " scale_fac " << scale_fac << "\n";
```

Comment:
Do you mean to have this print statement?

Reply:
oops. no. that was only to check if its called.
Will remove it

Reply:
This commit deletes the print statement 
7437cb0